jvm-packages/xgboost4j-example/src/test/resources/iris.csv

```diff
@@ -0,0 +1,150 @@
+5.1,3.5,1.4,0.2,Iris-setosa
```

Comment:
Please don't copy data in the git repository.

Reply:
@trivialfis  this file was not present in this repository at all. It is redownloaded every time for the integration tests. However, reusing this approach is not suitable for unittests :( I could consider rewriting examples to reuse existing datasets, but this would mean quite some rework :( 
I want to have the unittests present as testing that examples still work would require bildkite approval otherwise.

Reply:
Hi, thank you for sharing the consideration. I think we will have to leave the unittest out for this PR or write the unittest in a different way, like using existing datasets in XGBoost or generating a synthetic dataset.

Reply:
Hm, ok. I'll make sure that the unittests generate synthetic data which follows the iris dataset structure.

Reply:
Perhaps the example can use synthetic data instead?

Reply:
@trivialfis this would be quite a big change for this pull request. I'd like these to be done separately

Reply:
No need for now.